### PR TITLE
Add How do I use a Windows file with a Linux app?

### PR DESCRIPTION
**Pull request by EricGT**
Explains how to create a symbolic link to use a Windows file from an Linux app.

### CODE REVIEW

**Comment by bitcrazed** [comment]
### :white_check_mark: Validation status: passed


For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C1%5C26%5C5ef5d453-6fc9-df30-73f0-2cf3b60c84c3%5CPullRequest%5C201701261941331657-86%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: commandline/WSL/faq.md [hunk 1]
```diff
@@ -122,6 +123,28 @@ Mount points for hard drives on the local machine are automatically created and
...
+
+One of the main limitations of using WSL is that changing Linux files with a Windows app or tool is not allowed. See: [Do not change Linux files using Windows apps and tools](https://blogs.msdn.microsoft.com/commandline/2016/11/17/do-not-change-linux-files-using-windows-apps-and-tools/)
+
+However changing Windows files with a Linux app or tools is allowed.
+
+One way to use a Windows file with a Linux app is to use an absolute path, e.g. `/mnt/c/Users/<Windows User>/Documents/Projectes/<filename>`.
+
+However not all Linux apps or tools can access a file using `/mnt`.
+
+A solution is to create a [symbolic link](http://manpages.ubuntu.com/manpages/precise/man7/symlink.7.html).
```
Comment (bitcrazed) [inline commandline/WSL/faq.md]: Could I suggest a little re-wording for those who may be unfamiliar with symlinks:

A solution is to create a symbolic link (...) to map a virtual Linux folder to a physical Windows folder. For example: To map a `/home/<Linux user>/Projects` virtual folder to the physical `C:\Users\<Windows User>\Documents\Projects` Windows folder, execute the following command:

Comment (EricGT) [inline commandline/WSL/faq.md]: @bitcrazed 

Any changes you wish to make are welcome and appreciated. I just wanted a section in the WSL FAQ that noted the reason for needing to use symlinks and how to make one. It doesn't matter to me by who or how it gets done. :)

Do you need me to make the changes and resubmit or will you just make them and apply the change?

**Comment by jackchammons** [comment]
@EricGT Thanks for this. Really great info to be adding to our docs.

@bitcrazed I'll make that small change.

**Comment by EricGT** [comment]
Thanks. You guys are awesome.

**Comment by bitcrazed** [comment]
Not at all - thanks Eric - it's so awesome that you're passionate enough about this tech to spend your time helping fix our docs 😀

**Pull request merged**